modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -3,10 +3,11 @@
 // applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
 
 [id="odc-interacting-with-applications-and-components_{context}"]
-= Interacting with the application and the components
+= Interacting with applications and components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the *Developer* perspective of the web console provides the following options to interact with applications and components:
```

Comment:
This made the change from `*Topology* view` to `*Topology View*`, but the addition on line 10 uses it the old way of `*Topology* view`. Just wondering if it needs to be consistent throughout, or if "**Topology View**" is the exact term in the navigation panel, and "**Topology** view" is what is used when speaking of it generally?

Reply:
It must be **Topology view** on both Line 8 and 10

Reply:
Will make the change in the new commit.